lib/src/config_writer/vscode_config_writer.dart

```diff
@@ -33,34 +33,49 @@ class VscodeConfigWriter extends ConfigWriter {
   String writeConfig(String fileContent) {
     /// launch.json usually contains comments, which is valid only in JSON5.
     /// At this point however we cannot preserve these comments.
-    fileContent = fileContent.replaceAll(RegExp('.+//.+\n'), "");
```

Comment:
Any reason why this was removed?

Reply:
Yes, in my env config I have urls and this regex was corrupting the urls by removing the slashes in the url. Hence why I added the Try/Catch block.

Reply:
My bad about this
I think we can update the regex to catch comments only.

Reply:
Can you try out this package?
https://pub.dev/packages/json5

Reply:
Sure, I'll give it a shot